djangocms_versioning/helpers.py

```diff
@@ -306,3 +306,21 @@ def get_latest_admin_viewable_page_content(page, language):
     ).order_by(
         "versions__state"
     ).first()
+
+
+def get_list_display_config(model):
```

Comment:
Why does this helper exist? I can't see it used anywhere and no comments as to why it could exist.